### PR TITLE
Develop merge into Master

### PR DESCRIPTION
**Pull request by mrtizmoatwork**
Queuing this up so we can easily see what changes we are adding to master and can comment on them and revise before merging.

### CODE REVIEW

**Comment by mrjoshc** [comment]
@myhndl TODOS are done for Ambassador docs. This is ready for review and merge.

File: _i18n/en-US/_docs/development-environment-linux.md [hunk 1]
```diff
@@ -109,6 +107,7 @@ $ python
 
 You should be able to import `polyswarmclient` without issue.
 
-Next, we'll walk you through building your very own PolySwarm Microengine, capable of detecting the EICAR test file.
-
-[Make a "Hello World" Microengine →](/microengines-scratch-to-eicar/)
+Now that we have a working development environment let's move onto developing our PolySwarm participant:
+* [I want to develop a Microengine →](/microengines-scratch-to-eicar/)
+* [I want to develop an Ambassador →](TODO: link)
```
Comment (mrtizmoatwork) [inline _i18n/en-US/_docs/development-environment-linux.md]: @mrjoshc TODO ^

File: _i18n/en-US/_docs/development-environment-linux.md [hunk 2]
```diff
@@ -109,6 +107,7 @@ $ python
...
+* (coming soon) I want to develop an Arbiter (TODO: grey this out, don't link anywhere)
```
Comment (mrtizmoatwork) [inline _i18n/en-US/_docs/development-environment-linux.md]: @mrjoshc TODO ^

File: _i18n/en-US/_docs/microengines-scratch-to-eicar.md [hunk 3]
```diff
@@ -113,6 +120,8 @@ The `scan` function's `content` parameter contains the entire content of the art
 The following are 2 examples for how you can write your `scan()` function to detect `EICAR`.
 Update the code in your `__init__.py` file with the changes from one of these examples.
 
+<!-- TODO: technically both of these examples only catch 1 of many instances of EICAR -->
```
Comment (mrtizmoatwork) [inline _i18n/en-US/_docs/microengines-scratch-to-eicar.md]: @myhndl @mrjoshc what's with this TODO?

File: _i18n/en-US/_docs/microengines-scratch-to-eicar.md [hunk 4]
```diff
@@ -164,36 +172,37 @@ class Microengine(AbstractMicroengine):
...
 
-[Test Linux-based Engines →](/testing-linux/)
+## Testing Your Microengine
+
+Once everything is in place, let's test our microengine:
+
+[Test Linux-based participants →](/testing-linux/)
 
-[Test Windows-based Engines →](/testing-windows/)
+[Test Windows-based microengines →](/testing-windows/)
```
Comment (mrtizmoatwork) [inline _i18n/en-US/_docs/microengines-scratch-to-eicar.md]: @myhndl why does this say "microengines" when the linux one says "participants" ?

**Comment by mrtizmoatwork** [comment]
@mrjoshc @myhndl a couple more TODO's to resolve. Then it should be ready to publish.

**Comment by mrjoshc** [comment]
@mrtizmoatwork Good catch. My last push should resolve the link/aside TODOs.

**Pull request merged**